guide/popular-topics/permissions.md

```diff
@@ -1,8 +1,8 @@
 # Permissions
 
-Permissions are Discords primary feature enabling users to customize the workings of their server to their liking.
+Permissions are Discords primary feature, enabling users to customize the workings of their server to their liking.
```

Comment:
`Discords` is possessive so it should have an apostrophe.  Also unsure of usage of `server` vs `guild` throughout the guide, which is preferred.  I would say `guild` to be in agreement with the docs, but I haven't looked throughout.